bears/c_languages/CSecurityBear.py

```diff
@@ -8,7 +8,7 @@
         output_format='regex',
         output_regex=r'.+:(?P<line>\d+):(?P<column>\d+):\s*'
                      r'\[(?P<severity>\d)\]\s*'
-                     r'\((?P<origin>.+)\) (?P<message>.+)',
+                     r'\((?P<origin>.+)\) (?P<message>.+:.+)',
```

Comment:
let's rather make oigin non greedy, CC @Makman2 for help.

Also add explanation ideally as a comment.